app/views/shared/_contact_panels.html.erb

```diff
@@ -3,179 +3,211 @@
-            <% end %>
-
-            <% if display_chat_migration_message? %>
-              <li class="contact-panel__additional-info"><%= t('contact_panels.chat.offline_migration_html') %></li>
-            <% end %>
-          </ul>
-
-          <div class="contact-panel__button-container" id="js-chat-cta">
-            <div class="contact-panel__button button is-disabled t-chat-button">
-              <%= chat_opening_hours.call_to_action %>
+    <div class="l-contact-panels t-contact-panels <%= 'l-contact-panels--no-border-top' if contact_panels_border_top %> <%= 'l-contact-panels--homepage' if contact_panels_homepage %>">
+        <div class="l-contact-panel">
+            <div class="contact-panel contact-panel__chat-<%= chat_opening_hours.status %> t-chat" id="js-chat">
+                <%= heading_tag level: 2, class: 'contact-panel__heading' do %>
+                <span class="icon icon--web-chat"></span><%= @footer.web_chat.heading %>
+                <% end %>
+
+                <p class="contact-panel__chat-javascript t-chat-javascript">
+                    <%= t('contact_panels.chat.javascript.description_html') %>
+                </p>
+
+                <div class="contact-panel__dynamic-content is-loading" id="js-chat-content">
+                    <div class="contact-panel__preloader"></div>
+
+                    <p class="contact-panel__chat-description t-chat-description" id="js-chat-description">
+                        <%= chat_opening_hours.description %>
+                        <%= '*' if translation?('contact_panels.chat.smallprint') %>
+                    </p>
+
+                    <h3 class="contact-panel__opening-times-heading">Open:</h3>
+
+                    <ul class="t-chat-opening-times contact-panel__list unstyled-list">
+                        <% unless alternate_hours_active? %>
+                        <li class="contact-panel__additional-info"><%= @footer.web_chat.additional_one %></li>
+                        <li class="contact-panel__additional-info"><%= @footer.web_chat.additional_two %></li>
+                        <li class="contact-panel__additional-info"><%= @footer.web_chat.additional_three %></li>
+                        <% end %>
+
+                        <% if display_chat_migration_message? %>
+                        <li class="contact-panel__additional-info"><%= t('contact_panels.chat.offline_migration_html') %></li>
+                        <% end %>
+                    </ul>
+
+                    <div class="contact-panel__button-container" id="js-chat-cta">
+                        <div class="contact-panel__button button is-disabled t-chat-button">
+                            <%= chat_opening_hours.call_to_action %>
+                        </div>
+                    </div>
+                </div>
+
+                <p class="smallprint t-welsh-smallprint"><%= @footer.web_chat.small_print %></p>
```

Comment:
The content for this is currently missing (not as a result of this work). It appears to need ```t('contact_panels.chat.smallprint')``` here to pull in the text.

Reply:
Agreed that a new ticket will be created to fix this